.circleci/config.yml

```diff
@@ -30,16 +30,21 @@ jobs:
             # TODO: Replace with 'docker-compose run status'
             # Example: https://github.com/docker/compose/issues/374#issuecomment-310266246
             sleep 45
+
             # TODO: Replace with more organized BATS tests 'docker-compose run tests'
             # Example: https://github.com/StackStorm/stackstorm-ha/blob/master/tests/st2tests.sh
             docker-compose exec st2client st2 action list --pack=core
             docker-compose exec st2client st2 run core.local cmd=date
             docker-compose exec st2client st2 pack install github
             docker-compose exec st2client st2 execution list
       - run:
-          when: always
-          name: List created Services
-          command: docker-compose ps
+          when: on_fail
+          name: Troubleshooting the build failure
+          command: |
+            docker-compose ps
+            # Display logs to help troubleshoot build failures, etc
+            docker-compose logs --tail="500" st2api
```

Comment:
Moved the logs command to this dedicated section for better troubleshooting

Reply:
https://circleci.com/gh/StackStorm/st2-docker/2673 

with the first failure

Reply:
Yeah, I just added this as a quick thing to make troubleshooting easier (ideally we would tail all the containers, etc. on failure).

First I thought it may be issue in the changes itself, but now I think it was likely just a start up race in the tests - ideally we would use something more reliable than sleep, but in the worst case we can also bump sleep time a bit more.